app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -96,9 +97,10 @@
 import fr.free.nrw.commons.utils.DialogUtil;
 import fr.free.nrw.commons.utils.ExecutorUtils;
 import fr.free.nrw.commons.utils.LayoutUtils;
+import fr.free.nrw.commons.utils.LocationUtils;
```

Comment:
This import seems unused kindly remove it.

Reply:
Sure! Might have been added during merging conflicts. Removed now